pkg/model/clusterentity.go

```diff
@@ -17,7 +17,7 @@ type ClusterEntity struct {
 	RuntimeID  string            `db:"notNull"`
 	Runtime    *keb.RuntimeInput `db:"notNull"`
 	Metadata   *keb.Metadata     `db:"notNull"`
-	Kubeconfig string            `db:"notNull,encrypt"`
+	Kubeconfig string            `db:"notNull"`
```

Comment:
this kind of data should stay encrypted actually

Reply:
Thanks for spotting that 👍 I did that to test the dump locally and forgot to remove :(